serve

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+_ALL="en_US.UTF-8" \
+bundle exec jekyll serve \
```

Comment:
Thanks! The existing `Makefile` also runs some of the other asset generation stuff in parallell/in the background, so if we can keep all the stuff together I think we can make the project easier to start up for new folks